src/ansys/aedt/core/maxwell.py

```diff
@@ -3179,6 +3179,50 @@ def assign_resistive_sheet(
 
         return self._create_boundary(name, props, "ResistiveSheet")
 
+    def order_coil_terminals(self, winding_name=None, list_of_terminals=None):
+        """Order coil terminals
+
+        Create custom connection order amongst different turns in a Winding definition.
+        Note that this feature is only available in the A-Phi formulation.
+
+        Parameters
+        ----------
+        winding_name : str
+            Name of Winding in AEDT
+
+        list_of_terminals: lst of str
```

Comment:
```suggestion
        list_of_terminals: lst of str, optional.
```
Why did you put it to None? and if this is the case you are not handling in the code implementation the value None. The method will fail.

Reply:
@gmalinve the terminals can't be optional, and indeed it will raise an error. The method needs to get the list of terminals to be ordered.

Reply:
removed optional arguments